Sources/Extensions/UIKit/UIViewControllerExtensions.swift

```diff
@@ -44,5 +44,36 @@ public extension UIViewController {
 		NotificationCenter.default.removeObserver(self)
 	}
 	
+    /// SwifterSwift: Helper method to display an alert on any UIViewController subclass. Uses UIAlertController to show an alert
+    ///
+    /// - Parameters:
+    ///   - title: title of the alert
+    ///   - message: message/body of the alert
+    ///   - buttonTitles: (Optional)list of button titles for the alert. Default button i.e "OK" will be shown if this paramter is nil
+    ///   - highlightedButtonIndex: (Optional) index of the button from buttonTitles that should be highlighted. If this parameter is nil no button will be highlighted
+    ///   - completion: (Optional) completion block to be invoked when any one of the buttons is tapped. It passes the index of the tapped button as an argument
```

Comment:
Mind documenting the `Return` value and mentioning that is discardable? Otherwise 👍

Reply:
suggested changes incorporated in the latest commit.